docs/operate-a-stake-pool/overview.md

```diff
@@ -14,31 +14,41 @@ There are excellent guidelines available on how to set up cardano-node as a stak
 
 ## What if I don't meet the requirements?
+
 If you don't meet all of the qualifications, you'll need a strong desire to study and the understanding that you won't be able to learn everything in a few weeks.
 
 We observed people who had no prior knowledge of Linux, shells, or networking, but who were committed and had enough time to properly deal with it, and who now manage a profitable stake pool. It isn't for everyone, and it isn't going to be simple. Here are a few resources to get you started:
+
 - [Start playing around with Linux](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview).
 - [Have a look at nix and NixOS](https://nixos.org).
 - [Stake pool operator forum](https://forum.cardano.org/c/staking-delegation/156).
 
 ## Choose security over comfort
+
 Best practices should always be a key consideration when running a stake pool. Security isn't something you can turn on or off or change in a configuration file. It is both an attitude and a way of life, therefore consider following the [security related topics with stake pool operators](https://forum.cardano.org/c/staking-delegation/stake-pool-security/157) on the Cardano Forum. Make sure to fully understand [Cardano Key Pairs](cardano-key-pairs) in the basic category.
- 
+
 ## Learn the basics
-The basic category starts with an [introduction to Cardano](introduction-to-cardano), you will learn the [relay and block producer topology](stake-pool-networking), what the [hardware requirements](hardware-requirements) are, which [keys are available](cardano-key-pairs), which are hot and sensitive, and which you should never save on a server, no matter how convenient it is.  
+
+The basic category starts with learning about [relay and block producer topology](stake-pool-networking), what the [hardware requirements](hardware-requirements) are, which [keys are available](cardano-key-pairs), which are hot and sensitive, and which you should never save on a server, no matter how convenient it is.  
+
+:::tip info
+If you would like to do a deep dive on technical concepts please visit [Consensus](docs/get-started/technical-concepts/consensus-staking.md) and [Network & Communication](docs/get-started/technical-concepts/network-communication.md)
+:::
 
 ## Stake pool operator resources
-- [Guild Operators](https://cardano-community.github.io/guild-operators), famous for their [CNTools](https://cardano-community.github.io/guild-operators/#/Scripts/cntools) and top-notch content. 
+
+- [Guild Operators](https://cardano-community.github.io/guild-operators), famous for their [CNTools](https://cardano-community.github.io/guild-operators/#/Scripts/cntools) and top-notch content.
 - [Topology Updater](https://cardano-community.github.io/guild-operators/#/Scripts/topologyupdater) is intended to be a temporary solution to allow everyone to activate their relay nodes without having to postpone and wait for manual topology completion requests.
```

Comment:
can be removed as it is not recommended and required anymore in a fully deployed P2P network. 
I will sunset the TU service in Q3/4 this year.